roofit/roofitcore/src/RooVectorDataStore.cxx

```diff
@@ -1141,9 +1141,9 @@ void RooVectorDataStore::RealFullVector::Streamer(TBuffer &R__b)
      // WVE - It seems that ROOT persistence turns null pointers to vectors into pointers to null-sized vectors
```

Comment:
Maybe this comment is not valid anymore in case we move to unique_ptr ?

Reply:
I don't know, good question, I have not checked this yet. It's not a priority but maybe at some point we can check this.